Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -831,21 +810,20 @@
             "endTime": "16:00:00"
           }
         },
-        "Start": 20,
-        "Length": 15
+        "Start": 16,
+        "Length": 22
       }
     ]
   },
   {
-    "Input": "laten we elkaar van 13:30 tot 4 ontmoeten",
+    "Input": "laten we elkaar ontmoeten van 1:30 's middags tot 4",
```

Comment:
@iMicknl is this edit correct? Or should both be supported (assuming the previous was not a duplicate of another spec)?

Reply:
`13:30 tot 4` or `13:30 tot 16:00` makes more sense than 1:30 's middags in written language. However, for voice the latter would make more sense.

Reply:
Added spec "laten we elkaar ontmoeten van 13:30 tot 4"